java/iot3/mobility/build.gradle

```diff
@@ -20,12 +20,10 @@ repositories {
 }
 
 dependencies {
-    // This dependency is exported to consumers, that is to say found on their compile classpath.
-    //api 'com.hivemq:hivemq-mqtt-client:1.3.0'
-
-    // This dependency is used internally, and not exposed to consumers on their own compile classpath.
+    // Internal dependencies, not exposed to consumers on their own compile classpath.
     implementation 'org.json:json:20231013'
```

Comment:
Vulnerabilities from dependencies:
[CVE-2023-51074](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-51074)
Voir: https://mvnrepository.com/artifact/org.json/json/20240303

Reply:
Oups, la vulné concerne 20231013: https://mvnrepository.com/artifact/org.json/json/20231013.

Reply:
Updating the JSON library to version 20240303.